### ISSUE TITLE
rdctl factory-reset asks for sudo password via WSL

### ISSUE THREAD

**Issue by IsaSih** [rancher-sandbox/rancher-desktop#4539]
### Actual Behavior

`rdctl factory-reset` asks for sudo password when bats are running via WSL

### Steps to Reproduce

1. Start Rancher Desktop (installed build)
2. Run factory-rest.bats from a WSL distro (Ubuntu)
3. Verify that the sudo password is requested in the terminal

### Result

rdctl factory-reset asks for sudo password via WSL

### Expected Behavior

rdctl factory-reset must NOT ask for a sudo password as it does not perform any sudo commands

### Rancher Desktop Version

1.9.0-tech-preview

### Rancher Desktop K8s Version

N/A

### Which container engine are you using?

containerd (nerdctl)

### What operating system are you using?

Windows

### Operating System / Build Version

windows 11

### What CPU architecture are you using?

x64

### Linux only: what package format did you use to install Rancher Desktop?

None

**Comment by Nino-K** [comment]
I believe part of the problem is, we do this iptables [cleanup](https://github.com/rancher-sandbox/rancher-desktop/blob/24f14e6ffee485060a492e6811f87ac7bafe9a80/bats/tests/helpers/vm.bash#L37-L43) during factory reset on Windows that requires sudo.

**Comment by jandubois** [comment]
Yes, you are supposed to have enabled passwordless sudo in the distro you use to run BATS on Windows:

```console
jan@DESKTOP-4992680:~$ sudo cat /etc/sudoers.d/nopasswd
%sudo ALL=(ALL) NOPASSWD:ALL
```

#4529 mentions that this needs to be added to the README, so this issue might be closed as a duplicate.

However it looks like #4513 has been closed as a duplicate of this issue, which isn't correct because that issue is on Linux, not Windows, so should maybe be reopened for verification.
